### PR TITLE
Trim fat

### PR DESCRIPTION
**Pull request by jquense**

### CODE REVIEW

**Comment by jquense** [comment]
no proptypes!

**Comment by jquense** [comment]
no prop-types!

File: src/AbstractNav.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import qsa from 'dom-helpers/query/querySelectorAll';
 import PropTypes from 'prop-types';
-import elementType from 'prop-types-extra/lib/elementType';
+import { elementType } from 'prop-types-extra';
```
Comment (jquense) [inline src/AbstractNav.js]: switched to this style in order to have the import removed entirely for prod

**Comment by RKushnir** [comment]
Could you please release a new version with this change, as we're getting a warning because of the type mismatch for `outOfBoundaries`?

**Pull request merged**